pkgs/development/perl-modules/generic/builder.sh

```diff
@@ -4,7 +4,11 @@ PERL5LIB="$PERL5LIB${PERL5LIB:+:}$out/lib/perl5/site_perl"
 
 perlFlags=
 for i in $(IFS=:; echo $PERL5LIB); do
-    perlFlags="$perlFlags -I$i"
+    # exclude paths pointing to perl derivation (like /nix/store/abcdfghijklmnpqrsvwxyz0123456789-perl-5.28.2/lib/perl5/site_perl)
+    # they are needless pointing to correct perl, and source of subtle bugs pointing to another perl
+    if [[ ! $i =~ /[0123456789abcdfghijklmnpqrsvwxyz]{32}-perl-[0-9.]+/lib/perl5/site_perl$ ]] && [[ $perlFlags != *" -I$i"* ]]; then
```

Comment:
Maybe `${perl}/lib/perl5/site_perl` instead?

Reply:
Maybe, I need to check which `perl` is there during cross-compilation

Reply:
@FRidh, ok.
also a bug was found (regex syntax requires `sed -E`, not `sed -e`), so I fixed it and added additional logging what exactly is patched